services/identity/MailClient.ts

```diff
@@ -2,14 +2,20 @@ import axios from "axios"
 
 import logger from "@logger/logger"
 
-const { POSTMAN_API_KEY } = process.env
 const POSTMAN_API_URL = "https://api.postman.gov.sg/v1"
 
 class MailClient {
-  async sendMail(recipient: string, body: string): Promise<void> {
-    if (!POSTMAN_API_KEY)
+  private readonly POSTMAN_API_KEY: string
```

Comment:
Actually this commit is really a refactor commit than a test commit - in future, we might want to split out refactor commits into a separate PR from the test one